engine/runtime/src/main/java/org/enso/compiler/TreeToIr.java

```diff
@@ -1389,7 +1457,12 @@ private List<IR.Name> qualifiedNameSegments(Tree t, boolean generateId) {
   @SuppressWarnings("unchecked")
   IR$Module$Scope$Import translateImport(Tree.Import imp) {
     try {
-      Option<IR$Name$Literal> rename = Option.apply(imp.getAs()).map(as -> buildName(as.getBody(), true));
+      Option<IR$Name$Literal> rename;
+      if (imp.getAs() == null) {
+        rename = Option.empty();
+      } else {
+        rename = Option.apply(buildName(imp.getAs().getBody(), true));
+      }
```

Comment:
My personal preference is for the former, but when using Scala in Java both look ugly anyway.